### DISCUSSION TITLE
怪了，为什么纯蓝牙设备会离线

### DISCUSSION THREAD

**Discussion by NXY666** [XiaoMi/ha_xiaomi_home#505] [Q&A]
我的理解是按钮（蓝牙版）→中枢网关/小爱音箱→触发自动化

![Screenshot_20241230_170936_com_xiaomi_smarthome_OperationCommonWebViewActivity.jpg](https://github.com/user-attachments/assets/e84d9bb0-f2bf-4d84-aee3-2bf89f89bf89)

就是这玩意，纯蓝牙连的。

![Screenshot_20241230_171438_com_xiaomi_smarthome_SpecSceneEditPageV2.jpg](https://github.com/user-attachments/assets/8db39666-14f5-494b-ad2c-e8a3c4f64830)

米家显示未连接中枢，但是米家自动化能正常用。

![Screenshot_20241230_171630_io_homeassistant_companion_android_WebViewActivity.jpg](https://github.com/user-attachments/assets/0ed98168-9110-40f0-88d8-69988df0313f)

homeassistant内集成提示离线，所有传感器均不可用。

这就很奇怪了，没连接网关，那它蓝牙连哪去了？

**Comment by jjcszxh** [comment]
连中枢，中枢目前还是云端的，ptx的东西质量很差

**Reply by NXY666** [reply]
> 连中枢，中枢目前还是云端的，ptx的东西质量很差

我有点想不通，没连上蓝牙，它还能怎么和米家通信呢。

**Reply by jjcszxh** [reply]
> > 连中枢，中枢目前还是云端的，ptx的东西质量很差
> 
> 我有点想不通，没连上蓝牙，它还能怎么和米家通信呢。

你手机

**Reply by NXY666** [reply]
> 你手机

emm？所以我把手机蓝牙什么的关了，米家自动化（由这个蓝牙按钮触发）就无法成功执行吗？

但是好像蓝牙按钮仍然能成功触发自动化诶。
